src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+
+type Props = {
+  error: boolean;
+  deleteError: (value: boolean) => void;
+  errorMessage: string;
+};
+
+export const Notification: React.FC<Props> = ({
+  error,
+  deleteError,
+  errorMessage,
+}) => {
+  return (
+    <div
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal', {
```

Comment:
If you use `classnames` lib, you can pass all the classes as separate arguments: `classNames('notification', 'is-danger', ...)`. It's more readable I think.